opentreemap/treemap/templates/treemap/partials/user_favorite.html

```diff
@@ -0,0 +1,13 @@
+{% load util %}
+
+<a href="{{ favorite.map_feature|detail_link }}">
+    <div>
+        {% if favorite.photo %}
+            <img src="{{ favorite.photo }}" />
+        {% else %}
+            <div class="no-photo"><i class="icon-star"></i></div>
+        {% endif %}
+        <h4 class="common-name">{{ favorite.title }}</h4>
+        <h5>{{favorite.address}} - {{favorite.instance.name}}</h5>
```

Comment:
I switched to an em dash and am now hiding it if the address is falsey

![screen shot 2015-02-24 at 9 11 34 am](https://cloud.githubusercontent.com/assets/17363/6353227/49d28778-bc05-11e4-8fbe-928ad77be9b2.png)